geomesa-accumulo/geomesa-accumulo-datastore/pom.xml

```diff
@@ -179,6 +179,22 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
we should just need the groupId and artifactId here, right? seems like we could delete the rest since it's the same as the pluginManagement definition

Reply:
also, could you fix the indentation here? :D

Reply:
@elahrvivaz actually, we need this configuration. The one in the root pom does not work.

Reply:
Indentation fixed

Reply:
@elahrvivaz  - fixed